### PR TITLE
card cards detail

### PR DESCRIPTION
**Pull request by DaianaCarabetta**
A service folder was created with the call to the endpoint, the first components were created, card, cards and details.

### CODE REVIEW

**Comment by lautarob** [comment]
Please add some description to the PR.

File: vite-project/src/App.jsx [hunk 1]
```diff
@@ -1,5 +1,16 @@
+import Cards from "./components/Cards/Cards";
+import Detail from "./components/Detail/Detail";
+import { Route, Routes } from "react-router-dom";
+
 function App() {
-  return <></>;
```
Comment (lautarob) [inline vite-project/src/App.jsx:2]: Remove unnecessary line

File: vite-project/src/components/Card/Card.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const Card = ({ digimon }) => {
+  debugger;
```
Comment (lautarob) [inline vite-project/src/components/Card/Card.jsx]: Please don't push debuggers or this kind of things to the codebase.

File: vite-project/src/components/Cards/Cards.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React, { useEffect, useState } from "react";
+import Card from "../Card/Card";
+import { getDigimons } from "../../services/digimonService";
+
+const Cards = () => {
+  const [digimons, setDigimons] = useState(null);
+  debugger;
```
Comment (lautarob) [inline vite-project/src/components/Cards/Cards.jsx]: same here

File: vite-project/src/components/Cards/Cards.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  useEffect(() => {
+    getDigimons().then((data) => {
+      setDigimons(data);
+      debugger;
```
Comment (lautarob) [inline vite-project/src/components/Cards/Cards.jsx]: same here

File: vite-project/src/components/Cards/Cards.jsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    });
+  }, []);
+  debugger;
```
Comment (lautarob) [inline vite-project/src/components/Cards/Cards.jsx]: same here

File: vite-project/src/services/digimonService.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
+import axios from "axios";
+
+const URL = "https://digi-api.com/api/v1/digimon";
+const apiService = axios.create({ URL });
+
+export const getDigimon = async (name) => {
+  try {
+    const response = await apiService.get(`${URL}/${name}`);
+    debugger;
```
Comment (lautarob) [inline vite-project/src/services/digimonService.js]: same here

**Review by lautarob — CHANGES_REQUESTED** [review]

**Review by DaianaCarabetta — COMMENTED** [review]
debugger removed

**Pull request merged**